source/content/guides/drupal/drupal-hosted-md/05-dependencies.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
@stovak , still contains multiple references to D9.  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-md/dependencies

Reply:
same